### PR TITLE
🚀 Create 5 IOKs to detect different malware C2 panels

### PR DESCRIPTION
**Pull request by IlluminatiFish**
🟢 Additions:
- Created [`imbetter-stealer-1f52021a`](https://github.com/phish-report/IOK/commit/c8e6701ee623f00ad026192ddde071de34ecd859)
- Created [`amadey-botnet-afb0c86a`](https://github.com/phish-report/IOK/commit/5ce4bcb77d0136344805eca600350a7c11c927bc)
- Created [`lokibot-stealer-b5463607`](https://github.com/phish-report/IOK/commit/68688dbed94cc846f0a806136bdf8655d6447325)
- Created [`gomorrah-stealer-9bead31e`](https://github.com/phish-report/IOK/commit/ce5fa3307f216aa70bea02c49eacc0bb6d832973)
- Created [`rhadamanthys-stealer-26461dbb`](https://github.com/phish-report/IOK/commit/cb1ef52a23c0c7e637a923f5bc94d2c7bd4c5db1)

🟠 Changes:
- Change [`README`](https://github.com/phish-report/IOK/pull/182/commits/8e040e1866f931c5dd9e582a1686d95da5800ef3) to reflect new use case

Special thanks to [@ViriBack](https://twitter.com/ViriBack) for the feed of malware c2 panels catalogued [here](https://tracker.viriback.com)

### CODE REVIEW

**Review by actually-akac — APPROVED** [review]
These look good to me. Since this is a major change we might also wait for Bradley's approval.

**Review by bradleyjkemp — APPROVED** [review]
Looks good! Full support for this: a great use case for IOK

**Pull request merged**
